sw/airborne/modules/ctrl/ctrl_eff_sched_rot_wing.c

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (C) 2023 Dennis van Wijngaarden <D.C.vanWijngaarden@tudelft.nl>
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+/** @file "modules/ctrl/ctrl_eff_sched_rot_wing.c"
+ * @author Dennis van Wijngaarden <D.C.vanWijngaarden@tudelft.nl>
+ * The control effectiveness scheduler for the rotating wing drone type
+ */
+
+#include "modules/ctrl/ctrl_eff_sched_rot_wing.h"
+
+#include "firmwares/rotorcraft/stabilization/stabilization_indi.h"
+#include "modules/actuators/actuators_uavcan.h"
```

Comment:
This include drags the header `modules/uavcan/uavcan.h` which is part of the arch section. To avoid test compilation to fail, a dummy file should be added in `${PAPARAZZI_SRC}/tests/modules/test_arch`

Reply:
Changed it to an ABI message. No more need to include arch files.